Configurations/windows-makefile.tmpl

```diff
@@ -118,7 +118,7 @@ INSTALLTOP_dev={- # $prefix is used in the OPENSSLDIR perl snippet
 INSTALLTOP_dir={- $prefix_dir -}
 OPENSSLDIR_dev={- #
                   # The logic here is that if no --openssldir was given,
-                  # OPENSSLDIR will get the value from $prefix plus "/ssl".
+                  # OPENSSLDIR will get the value "$win_commonroot\\SSL".
```

Comment:
When I saw this first thing in my mind was "there is no $win_commonroot environment variable in Windows." Then of course I figured out that $win_commonroot is reference to variable in template in question. But this calls for question if this is best way to formulate this commentary. Maybe `%CommonProgramFiles%\SSL` would be better for Windows people who are trying to make sense of it... This is kind of side note... In sense that I really don't expect changes, unless others agree...